idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -1893,11 +2070,11 @@ def initialize(blk, gas_phase_state_args=None, solid_phase_state_args=None,
                             Constraint, descend_into=False):
                         c.activate()
 
-            blk.bubble.reactions.initialize(outlvl=outlvl,
+            blk.bubble.reactions.initialize(outlvl=idaeslog.DEBUG,
```

Comment:
Do not hard code the output level - use the `outlvl` argument provided when initialization is called.